core/domain/general_feedback_services.py

```
@@ -238,6 +255,137 @@ def create_lesson_feedback(
+
+    Args:
+        feedback_id: str. ID of the LessonFeedbackModel to update.
+        new_status: str. The new status value. Must be a valid status choice.
+        exp_id: str. The exploration from which the feedback is being
+            accessed.
+
+    Returns:
+        Optional[LessonFeedback]. The updated report, or None if not found.
+    """
+    if new_status not in feconf.STATUS_CHOICES:
+        raise ValueError('Invalid status: %s' % new_status)
```

Ruff (0.16.0)

[warning] Use format specifiers instead of percent format

Replace with format specifiers

(UP031)